rqt_rviz/src/rqt_rviz/rviz.cpp

```diff
@@ -85,7 +85,7 @@ void RViz::initPlugin(qt_gui_cpp::PluginContext& context)
   menu_bar->setVisible(!hide_menu_);
   widget_->setMenuBar(menu_bar);
 
-  widget_->initialize(display_config_.c_str());
```

Comment:
This breaks passing in the display config file as a command line argument (https://github.com/ros-visualization/rqt_robot_plugins/blob/master/rqt_rviz/scripts/rqt_rviz https://github.com/ros-visualization/rqt_robot_plugins/blob/d86361f32659b4620e7c2198f826857800ad0554/rqt_rviz/src/rqt_rviz/rviz.cpp#L164).